Allow the Nunez family to pack up and move to Tacoma, not just talk about it.

I-am-Erk opened the pull request:
#### Summary
Content "Allow the Nunez family to move to Tacoma Commune if you help them set it up."
<!-- This section should consist of exactly one line, edit the one above.
1. Replace the word "Category" with one of these words: Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N.
2. Replace the text inside the quotes with a brief description of your changes.
Or if you don't want a changelog entry, replace the whole line with just the word "None" (with no quotes).
For more on the meaning of each category, see:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md
If approved and merged, your summary will be added to the project changelog:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt -->

#### Purpose of change
The Nunez family (Dana and Pablo) in the refugee center have a plot line about moving to the Tacoma Commune, but once finished they just stay at the refugee center. We now have the code to fix that.

#### Describe the solution
- [x] Add new terrain for a special "dana's bakery" building, trying out making custom building art for increased immersion
- [x] Add mapgen settings to add Dana's Bakery
- [x] Add EoC's to add Dana's Bakery a few days after you set things up for them
- [x] Teleport Dana and Pablo to the bakery
- [x] Finalize Dana and Pablo's dialogue in the refugee center
- [x] Add new dialogue for Dana in the commune
- [x] Add new dialogue for Pablo in the commune

In order to change their gear and things, it turned out to be easier just to delete the old Dana and Pablo, and create new ones on site at Tacoma.

This adds a new custom building (the parts can be repurposed to make other bullet trailers) that Dana and Pablo live in on Tacoma. It adds an EoC to move them there and delete them from the original map. It switches their topics and variables to be agnostic so that a person that is, technically, a new NPC will remember what you talked to the old ones about. I checked over their old missions and fixed some bugs that made it possible for the items to not spawn. I added a bit more of a clue to the tacoma foreman about the possibility of recruiting Pablo and Dana.

There are two stub-missions here that Pablo can assign you that cannot, at this time, be completed nor assigned. At the time of this writing this PR is pushing close to 1000 lines, and I have more to go before I am done, so I am gonna leave those there, and make them assignable in a f/u PR that also adds the complete mission.

#### Describe alternatives you've considered
They could just have a space in an existing building, or a simpler temporary shelter. I am not that guy.

#### Testing
This was a terrible slog to finish as evidenced by the PR completion date, and has more or less motivated me to see if it's possible to rewrite the dialogue code to be more human-useable, it's been a long time coming.

I can make it through all the steps of this quest line but it really needs someone other than me to try it out and make sure I am not missing anything. To get to tacoma you must complete the first two quests from Smokes, then the third will introduce you to the ranch leader. After that, talking to Pablo will initiate this quest

#### Additional context
Maleclypse did a huge amount of the work keeping this going and deserves a lot of credit for this. I would have abandoned it a long time ago out of frustration without Mal's efforts.

Quick mockup of the building: Let's hope this works!
![image](https://user-images.githubusercontent.com/45136638/166590761-09dea879-203f-4b4a-9743-0b15d5c306e8.png)

I-am-Erk committed:
start moving the nunez family

I-am-Erk committed:
Finalize terrain IDs

I-am-Erk committed:
Create bullet_trailer.json

I-am-Erk committed:
Update bullet_trailer.json

I-am-Erk committed:
Update bullet_trailer.json

I-am-Erk committed:
Update Arrival_code.json

I-am-Erk committed:
Update Arrival_code.json

I-am-Erk committed:
Update Arrival_code.json

PatrikLundell commented:
I'm struck by the complete lack of actually bakery in Dana's Bakery...

If possible, I'd suggest an appliance filled building for the actual bakery, as that would showcase the use of appliances. A couple of ovens (one is probably enough if she's working alone), some kind of racks for rising bread and cooling loaves, a food processor, a dish washer, a fridge, a counter for the manual kneading/loaf forming stage, some large pots (casseroles?) for rising dough, rolling pin, etc.. with a compact area for the couple (double bed, toilet, shower, some basic furniture).

I-am-Erk commented:
> I'm struck by the complete lack of actually bakery in Dana's Bakery...
> 
> If possible, I'd suggest an appliance filled building for the actual bakery, as that would showcase the use of appliances. A couple of ovens (one is probably enough if she's working alone), some kind of racks for rising bread and cooling loaves, a food processor, a dish washer, a fridge, a counter for the manual kneading/loaf forming stage, some large pots (casseroles?) for rising dough, rolling pin, etc.. with a compact area for the couple (double bed, toilet, shower, some basic furniture).

The bakery starts out as the trailer kitchen because that's how they arrive at Tacoma. Later they'll begin building an actual bakery.

PatrikLundell commented:
That's reasonable, but I don't see anything looking like a kitchen. The closest thing I see is the sink. However, I like both the idea and that there's a progression.

I-am-Erk commented:
I may add some contextual item sprites to show the portable oven Dana is using, but otherwise, dialogue will explain how she's baking in there.

I-am-Erk committed:
copy in some of my dialogue for pablo

I-am-Erk committed:
some more pablo dialogue

I-am-Erk committed:
dana dialogue

I-am-Erk committed:
Update NPC_Dana_Tacoma.json

I-am-Erk commented:
Reopening, I'll see if I can polish this off. It's not far from done.

I-am-Erk committed:
Merge branch 'CleverRaven:master' into Get-your-own-darn-bakery-you-freeloafers

I-am-Erk committed:
Add dialogue flag related to Dana's pregnancy

allows you to ask about it later.

I-am-Erk committed:
Update NPC_Dana_Tacoma.json

This leaves her as a bit of a stub character, she should get some of her old conversation options back.

I-am-Erk committed:
Update NPC_Pablo_Tacoma.json

Improve dialogue conditionals, add stub missions. Needs dialogue to turn in the missions.

I-am-Erk committed:
Update Arrival_code.json

I-am-Erk committed:
Get ready to teleport NPCs

Gonna need a little more here, and the dialogue needs to be set up so that they use the right dialogue after moving.

I-am-Erk committed:
Update data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

I-am-Erk committed:
Update refugee dana

Make her stats fixed and change some her bread sale dialogue to be agnostic so that she can offer bread in Tacoma as well.

need to review her mission dialogue and make sure it works.

I-am-Erk committed:
refugee dana mission updates

took the mission out of the default framework so that it can be offered by tacoma-dana as well.

I-am-Erk committed:
Update NPC_Dana_Tacoma.json

Add the last of the baking topic. Set up her skills and stats.

I-am-Erk committed:
Update NPC_Pablo_Nunez.json

I-am-Erk committed:
Update NPC_Pablo_Tacoma.json

new gear and definition

I-am-Erk committed:
Update Arrival_code.json

integrate npc removal

I-am-Erk committed:
Update bullet_trailer.json

lint

I-am-Erk committed:
Update NPC_Pablo_Nunez.json

lint

I-am-Erk committed:
Update Arrival_code.json

lint

I-am-Erk committed:
Update NPC_Dana_Tacoma.json

lint

I-am-Erk committed:
Update NPC_Pablo_Tacoma.json

lint

I-am-Erk committed:
Update bullet_trailer.json

I-am-Erk committed:
Update bullet_trailer.json

I-am-Erk committed:
Update bullet_trailer.json

I-am-Erk committed:
Update bullet_trailer.json

symbols

I-am-Erk committed:
Update bullet_trailer.json

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
+    "color": "light_gray",
+	"looks_like": "t_window"
```
```suggestion
	"looks_like": "t_window",
```

Maleclypse committed:
Update data/json/furniture_and_terrain/special_use/bullet_trailer.json

Maleclypse committed:
Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "description": "A section of flooring made out of a tough, smooth material, colored in an atrocious mottled brown.",
+    "symbol": ".",
+    "color": "brown",
+    "bgcolor": "black",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "description": "A smooth wall of gleaming chrome, slightly dusty from the road.",
+    "flags": [ "SUPPORTS_ROOF", "WALL", "REDUCE_SCENT", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_nwall",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_nwall_window",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_nwall_eastcorner",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_wwall",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_ewall",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_inner_westcorner",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "id": "t_bullettrailer_inner_T",
+    "copy-from": "t_bullettrailer_nwall_westcorner",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "bullet trailer exterior wall",
+    "description": "A smooth wall of gleaming chrome, slightly dusty from the road.",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_swall_windows",
+    "symbol": "+",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_swall_door",
+    "symbol": "-",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_swall_door_o",
+    "symbol": "/",
+    "bgcolor": "black",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_swall_wheels",
+    "symbol": "o",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_swall",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_swall_eastcorner",
+    "symbol": "LINE_OXOX",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "move_cost_mod": -1,
+    "description": "A small trailer table, designed to convert into a bed when not in use.",
+    "symbol": "T",
+    "bgcolor": "brown",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "furniture",
+    "id": "f_bullettrailer_table_middle",
+    "symbol": "T",
+    "bgcolor": "brown",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "type": "furniture",
+    "id": "f_bullettrailer_table_bottom",
+    "symbol": "T",
+    "bgcolor": "brown",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "trailer cabinets",
+    "move_cost_mod": 0,
+    "symbol": "n",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "trailer sink",
+    "move_cost_mod": 0,
+    "symbol": "u",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "mini desk",
+    "move_cost_mod": 0,
+    "symbol": "t",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "trailer bathroom door",
+    "move_cost_mod": -1,
+    "symbol": "|",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "trailer bathroom door",
+    "move_cost_mod": 0,
+    "symbol": "/",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "name": "travel toilet",
+    "description": "A tiny plastic toilet for use in a travel trailer.  Unlike a normal toilet, it doesn't store water locally for you to siphon out.",
+    "symbol": "b",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "id": "f_bullettrailer_minisofa",
+    "name": "trailer sofa",
+    "symbol": "h",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "id": "f_bullettrailer_bed_foot",
+    "name": "trailer bed",
+    "symbol": "H",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "id": "f_bullettrailer_bed_head",
+    "name": "trailer bed",
+    "symbol": "H",
+    "bgcolor": "dark_gray",
```
```suggestion
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,313 @@
+    "id": "f_bullettrailer_sign_danabakery",
+    "name": "plywood sign for 'Dana's Bakery'",
+    "symbol": "#",
+    "bgcolor": "brown",
```
```suggestion
```

Maleclypse committed:
Apply suggestions from code review

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,284 @@
+  {
+    "type": "furniture",
+    "id": "f_bullettrailer_inner_door",
+    "name": "trailer bathroom door",
```
```suggestion
    "name": "trailer bathroom door",
    "description": "A small door heading to the bathroom.  Currently closed.",
```

Maleclypse replied:
```suggestion
    "name": "trailer bathroom door",
    "description": "A gray textured door, rimmed in chrome, like a very tacky version of a ship's hatch.",
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,284 @@
+  {
+    "type": "furniture",
+    "id": "f_bullettrailer_inner_door_o",
+    "name": "trailer bathroom door",
```
```suggestion
    "name": "trailer bathroom door",
    "description": "A small door heading to the bathroom.  Currently open.",
```

Maleclypse committed:
Update data/json/furniture_and_terrain/special_use/bullet_trailer.json

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,285 @@
+    "id": "f_bullettrailer_inner_door",
+    "name": "trailer bathroom door",
+    "description": "A gray textured door, rimmed in chrome, like a very tacky version of a ship's hatch.",
+    "move_cost_mod": -1,
```
```suggestion
    "move_cost_mod": -1,
    "required_str": -1,
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,285 @@
+    "type": "furniture",
+    "id": "f_bullettrailer_inner_door_o",
+    "name": "trailer bathroom door",
+    "move_cost_mod": 0,
```
```suggestion
    "move_cost_mod": 0,
    "required_str": -1,
```

Maleclypse committed:
Apply suggestions from code review

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    "copy-from": "f_desk"
+  },
+  {
+    "type": "furniture",
+    "id": "f_bullettrailer_inner_door",
```
```suggestion
    "type": "terrain",
    "id": "t_bullettrailer_inner_door",
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_bullettrailer_inner_door_o",
```
```suggestion
    "type": "terrain",
    "id": "t_bullettrailer_inner_door_o",
```

Maleclypse committed:
Apply suggestions from code review

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    "type": "furniture",
+    "id": "f_bullettrailer_desk",
+    "name": "mini desk",
+    "move_cost_mod": 0,
```
```suggestion
    "move_cost": 40,
```

Maleclypse committed:
Update data/json/furniture_and_terrain/special_use/bullet_trailer.json

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    "type": "furniture",
+    "id": "f_bullettrailer_desk",
+    "name": "mini desk",
+    "move_cost": 40,
```
```suggestion
    "move_cost_mod": 20,
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    "id": "t_bullettrailer_inner_door",
+    "name": "trailer bathroom door",
+    "description": "A gray textured door, rimmed in chrome, like a very tacky version of a ship's hatch.",
+    "move_cost_mod": -1,
```
```suggestion
    "move_cost": -1,
```

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    "type": "terrain",
+    "id": "t_bullettrailer_inner_door_o",
+    "name": "trailer bathroom door",
+    "move_cost_mod": 0,
```
```suggestion
    "move_cost": 0,
```

Maleclypse committed:
Apply suggestions from code review

Maleclypse commented on data/json/furniture_and_terrain/special_use/bullet_trailer.json:
```diff
@@ -0,0 +1,287 @@
+    "name": "trailer bathroom door",
+    "description": "A gray textured door, rimmed in chrome, like a very tacky version of a ship's hatch.",
+    "move_cost": -1,
+    "required_str": -1,
```
```suggestion
```

Maleclypse commented on data/json/npcs/refugee_center/surface_refugees/NPC_Dana_Nunez.json:
```diff
@@ -400,13 +412,50 @@
+    "type": "talk_topic",
+    "id": "TALK_DANA_Sourdough3",
+    "dynamic_line": "*takes the jar from you eagerly, studying it with care.  \"He's been through a lot, hasn't he.  Poor guy.  Aw man.\"  She turns back to you, beaming.  \"Oh, you will not regret this.  He may not look like much, but he's got it where it counts, kid.  It's gonna take me a few days to revive him after being abandoned so long, but he'll be operational soon and then you can come b ...
+    "speaker_effect": { "effect": [ { "u_sell_item": "landough_calrisean" }, { "finish_mission": "MISSION_REFUGEE_Dana_Sourdough" } ] },
```
```suggestion
    "speaker_effect": { "effect": [ { "u_sell_item": "landough_calrisean" }, { "finish_mission": "MISSION_REFUGEE_Dana_Sourdough", "success": true } ] },
```

Maleclypse committed:
Apply suggestions from code review

Maleclypse committed:
Lint

GuardianDll commented on data/json/npcs/refugee_center/surface_refugees/NPC_Pablo_Nunez.json:
```diff
@@ -226,7 +236,23 @@
+          ]
+        },
+        {
+          "arithmetic": [ { "global_val": "var", "var_name": "nunez_arrived", "type": "switch" }, "=", { "const": "0" } ]
```
```suggestion
          "math": [ "nunez_arrived", "=", "0"  ]
```

GuardianDll commented on data/json/npcs/tacoma_ranch/Nunez/Arrival_code.json:
```diff
@@ -0,0 +1,104 @@
+      { "mapgen_update": "tacoma_nunez_trailer_arrive", "om_terrain": "ranch_camp_55" },
+      { "mapgen_update": "refcenter_nunez_departure", "om_terrain": "evac_center_7" },
+      {
+        "arithmetic": [ { "global_val": "var", "var_name": "nunez_arrived", "type": "switch" }, "=", { "const": "1" } ]
```
```suggestion
        "math": [ "nunez_arrived", "=", "1"  ]
```

Maleclypse committed:
Apply suggestions from code review

Co-authored-by: Anton Simakov <67688115+GuardianDll@users.noreply.github.com>

GuardianDll commented on data/json/npcs/tacoma_ranch/Nunez/Arrival_code.json:
```diff
@@ -0,0 +1,104 @@
+    "condition": {
+      "compare_int": [ { "global_val": "var", "var_name": "nunez_travel", "type": "timer" }, "<", { "time_since_cataclysm": "days" } ]
+    },
+    "deactivate_condition": { "compare_int": [ { "global_val": "var", "var_name": "nunez_arrived", "type": "switch" }, "=", { "const": "1" } ] },
```
```suggestion
    "deactivate_condition": { "math": [ "nunez_arrived",  "==", "1" ] },
```

Maleclypse committed:
Update NPC_Pablo_Nunez.json

Maleclypse committed:
Update data/json/npcs/tacoma_ranch/Nunez/Arrival_code.json

Co-authored-by: Anton Simakov <67688115+GuardianDll@users.noreply.github.com>

Maleclypse committed:
Update Arrival_code.json

Maleclypse commented on data/json/npcs/tacoma_ranch/Nunez/NPC_Dana_Tacoma.json:
```diff
@@ -0,0 +1,187 @@
+  {
+    "type": "talk_topic",
+    "id": "TALK_TACOMA_Dana_1",
+    "dynamic_line": "*stands in the middle of a cramped kitchen, covered in flour.  The scene looks reminiscent of easier times, but the illusion is broken as he apron shifts to reveal a heavy machete and holstered pistol on  her belt.  She looks up as you approach, then beams in recognition. \"Welcome to Dana's Bakery, stranger!\"  She laughs brightly.  \"Glad you made out this way.  What can I  ...
```
```suggestion
    "dynamic_line": "*stands in the middle of a cramped kitchen, covered in flour.  The scene looks reminiscent of easier times, but the illusion is broken as he apron shifts to reveal a heavy machete and holstered pistol on  her belt.  She looks up as you approach, then beams in recognition.  \"Welcome to Dana's Bakery, stranger!\"  She laughs brightly.  \"Glad you made out this way.  What can I do for you?\"",
```

Maleclypse commented on data/json/npcs/tacoma_ranch/Nunez/NPC_Dana_Tacoma.json:
```diff
@@ -0,0 +1,187 @@
+  {
+    "type": "talk_topic",
+    "id": "TALK_TACOMA_Dana_Trailer3",
+    "dynamic_line": "*sighs and nods.  \"Making do with what I've got, but yeah. I can only crank out a couple loaves a day, and can't do anything fun.  Pablo has been talking about setting me up a separate kitchen to really get going, but we've got so much critical work to do, and fresh bread is a luxury, even if it's great for morale.\"",
```
```suggestion
    "dynamic_line": "*sighs and nods.  \"Making do with what I've got, but yeah.  I can only crank out a couple loaves a day, and can't do anything fun.  Pablo has been talking about setting me up a separate kitchen to really get going, but we've got so much critical work to do, and fresh bread is a luxury, even if it's great for morale.\"",
```

Maleclypse committed:
Apply suggestions from code review

Maleclypse commented:
@I-am-Erk data/json/npcs/tacoma_ranch/Nunez/NPC_Pablo_Tacoma.json you need to define responses for these talk_topics.   But that's probably the last thing needed for this PR.

I-am-Erk committed:
Update NPC_Pablo_Tacoma.json

Add missing responses

I-am-Erk committed:
Merge remote-tracking branch 'upstream/master' into Get-your-own-darn-bakery-you-freeloafers

I-am-Erk committed:
several small fixes

I-am-Erk committed:
further small fixes

I-am-Erk committed:
almost all the errors now, except the dang cabinet

I-am-Erk committed:
Fixes, then playtest fixes

Fixed some issues with how the bakery was working, made sourdough easier to get

I-am-Erk committed:
I.... I think it's working!

I-am-Erk committed:
Update Arrival_code.json

I-am-Erk committed:
Add a bit of a clue to the tacoma foreman

I-am-Erk committed:
linting

I-am-Erk committed:
hair change fix

Maleclypse commented on data/json/npcs/tacoma_ranch/Nunez/NPC_Pablo_Tacoma.json:
```diff
@@ -0,0 +1,315 @@
+  {
+    "type": "talk_topic",
+    "id": "TALK_TACOMA_Pablo_Bakery4",
+    "dynamic_line": "Not where you might think.\"  He sets aside his ledger and pulls out a page of bullet-point notes written in neat block letters. \"Everyone is on board with the idea and the amount of actual building to get it online is small enough that the carpenters are willing to help out, so I don't need any assistance with design.  For the really short term I basically just need materia ...
```
```suggestion
    "dynamic_line": "Not where you might think.\"  He sets aside his ledger and pulls out a page of bullet-point notes written in neat block letters.  \"Everyone is on board with the idea and the amount of actual building to get it online is small enough that the carpenters are willing to help out, so I don't need any assistance with design.  For the really short term I basically just need materials.  I've been collecting spare wood, but to get a good wood-fired oven I need to secure a source of firebrick.  In the longer run, Dana knows the principles of milling, but I need a resource book if I want to set up a functioning mill, first a muscle-powered one and then a windmill for long term.  I wish I'd thought of asking Jenny.  She's a mechanical engineer, but she would probably know how to get me started at least.  I'm also still looking into how to make a good vermin-proof cellar.  Longer term, I will need to work my way around our reliance on sugar, salt, and vinegar from before the cataclysm, but our stocks of those will hold out for months while I sort things out.",
```

Maleclypse committed:
Update data/json/npcs/tacoma_ranch/Nunez/NPC_Pablo_Tacoma.json

I-am-Erk commented:
Now we just need to convince someone to play test this some more. I'm sure I missed some stuff
